test/v0.16/NameConflictTest.thrift

```diff
@@ -0,0 +1,124 @@
+// Naming testcases, sepcifically for these tickets (but not limited to them)
```

Comment:
should this file also add the Apache License header?

Reply:
good catch, thanks